packages/wdio-cli/src/commands/config.ts

```diff
@@ -78,8 +78,9 @@ const runConfig = async function (useYarn: boolean, yes: boolean, exit = false)
                 }
             }
 
+            fs.ensureDirSync(path.join(process.cwd(), 'test'))
             await fs.promises.writeFile(
-                path.join(process.cwd(), 'tsconfig.json'),
+                path.join(process.cwd(), (hasFile('tsconfig.json') ? 'test' : ''), 'tsconfig.json'),
```

Comment:
I am seeing `hasFile('tsconfig.json')` here and in line 186, let's create a variable so we only need to call this function once.

Reply:
@christian-bromann  will do that 🥳